test/topology/test_mv.py

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Tests for materialized views that need a three-node cluster.
+Note that most materialized-view functional tests can make due with a single
+node, and belong cql-pytest. We also have topology_experimental_raft/
+test_mv_tablets.py for tests that each needs a cluster of a different
+size, and/or special Scylla parameters.
+"""
+import asyncio
+import logging
+import pytest
+
+from contextlib import asynccontextmanager
+from test.pylib.util import unique_name
+
+logger = logging.getLogger(__name__)
+
+@asynccontextmanager
+async def new_test_keyspace(cql, opts):
+    """
+    A utility function for creating a new temporary keyspace with given
+    options. It can be used in a "async with", as:
+        async with new_test_keyspace(cql, '...') as keyspace:
+    """
+    keyspace = unique_name()
+    await cql.run_async("CREATE KEYSPACE " + keyspace + " " + opts)
+    try:
+        yield keyspace
+    finally:
+        await cql.run_async("DROP KEYSPACE " + keyspace)
+
+previously_used_table_names = []
+@asynccontextmanager
+async def new_test_table(cql, keyspace, schema, extra=""):
+    """
+    A utility function for creating a new temporary table with a given schema.
+    Because Scylla becomes slower when a huge number of uniquely-named tables
+    are created and deleted (see https://github.com/scylladb/scylla/issues/7620)
```

Comment:
https://github.com/scylladb/scylla/issues/7620 was fixed

Reply:
I know, but 1. Only with Raft (though I guess the no-Raft case isn't interesting any more...) and 2. I never checked it was really fixed, and the workaround is very simple.